source/operations/install-deploy-manage/migrate-fs-gateway.rst

```diff
@@ -0,0 +1,203 @@
+   If the existing standalone system points to the root of the drive, you must use a separate drive for the new deployment.
+
+   Set the port to a custom point different than the existing standalone deployment.
+
+#. Add an alias for the new deployment with :mc:`mc alias set`
+ 
+   .. code-block:: shell
+      :class: copyable
+      
+      mc alias set PATH ACCESSKEY SECRETKEY
+
+   - Replace PATH with the IP address or hostname and port for the new deployment.
+   - Replace ``ACCESSKEY`` and ``SECRETKEY`` with the credentials you used when creating the new deployment.
+
+#. Import **configurations** from existing standalone deployment to new deployment
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin config import ALIAS < config.txt
+
+   - Replace ``ALIAS`` with the alias for the new deployment.
+
+#. Restart the server for the new deployment
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin service restart ALIAS
+   
+   - Replace ``ALIAS`` with the alias for the new deployment.
+   
+#. Duplicate **buckets** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc ls` with the ``--json`` flag to retrieve a list of the buckets that exist on the standalone deployment.
+   Use the list to recreate the buckets on the new deployment.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc ls ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+#. *(Optional)* Duplicate **tiers** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin tier ls` with the ``--json`` flag to retrieve a list of the tiers that exist on the standalone deployment.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin tier ls ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+   
+   Use the list to recreate the tiers on the new deployment.
+
+#. Duplicate **policies** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin policy list` with the ``--json`` flag  to retrieve a list of policies that exist on the standalone deployment.
+   
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin policy list ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+   Use the list to recreate the policies on the new deployment.
+
+#. Duplicate **groups** from existing standalone deployment to new deployment
+
+   Use :mc-cmd:`mc admin group list` with the ``--json`` flag to retrieve a list of groups that exist on the standalone deployment.
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc admin group list ALIAS --json
+
+   - Replace ``ALIAS`` with the alias for the existing standalone deployment.
+
+   Use the list to recreate the groups on the new deployment.
+
+#. Duplicate **users** from existing standalone deployment to new deployment
```

Comment:
We should note that if the original deployment was using external IAM, they need to recreate those settings in the SNSD deployment along with all associated policies.

Reply:
👍